helm/values-uat.yaml

```diff
@@ -96,10 +96,12 @@ microservice-chart:
     NODO_READ_TIMEOUT: "10000"
     NODO_CONNECTION_TIMEOUT: "10000"
     CARTS_MAX_ALLOWED_PAYMENT_NOTICES: "5"
-    ELASTIC_APM_SERVICE_NAME: pagopa-ecommerce-payment-requests-service
-    ELASTIC_APM_APPLICATION_PACKAGES: it.pagopa.ecommerce.payment.requests
-    ELASTIC_APM_SERVER_URL: http://quickstart-apm-http.elastic-system.svc:8200
-    ELASTIC_APM_LOG_SENDING: "true"
+    OTEL_RESOURCE_ATTRIBUTES: "service.name=pagopa-ecommerce-payment-requests-service,deployment.environment=uat"
```

Comment:
Modify also prod helm values

Reply:
Solved in https://github.com/pagopa/pagopa-ecommerce-payment-requests-service/pull/61/commits/54cc0848ef669df342967edaf655dbacd0a6b790